test/experimental_apis/web.js

```diff
@@ -129,6 +129,19 @@ describe('Experimental Web API', () => {
       return should(publicNames).containDeep(['safe://_publicNames#thisIsATestDomain']);
     });
 
+    it('should add publicName even on top of container old data format', async () => {
```

Comment:
@bochaco from you explanation, is this test not reliant on there being _no_ webId data already created?

I'm not sure how we can know that there is no RDF there already from this test...

I think to be truly sure, we need a func to be clearing the `Mock Vault` in between tests, or?

Reply:
Yes, probably, but I can jut simply create a new safeApp for that instance, no necessary to remove mock as the _publicNames container is per account, so with a new safeApp we know it's empty